ecs_fargate/datadog_checks/ecs_fargate/ecs_fargate.py

```diff
@@ -119,18 +119,33 @@ def check(self, instance):
             self.log.warning(msg, exc_info=True)
 
         for container_id, container_stats in stats.iteritems():
+            if container_stats is None:
+                self.log.debug("Could not collect stats from {}".format(container_id))
+                continue
+
             tags = container_tags[container_id]
 
-            # CPU metrics
+            #CPU metrics
             cpu_stats = container_stats.get('cpu_stats', {})
 
-            value = cpu_stats.get('system_cpu_usage')
+            value_system = cpu_stats.get('system_cpu_usage')
             if value is not None:
-                self.rate('ecs.fargate.cpu.system', value, tags)
+                self.rate('ecs.fargate.cpu.system', value_system, tags)
 
-            value = cpu_stats.get('cpu_usage', {}).get('total_usage')
+            value_total = cpu_stats.get('cpu_usage', {}).get('total_usage')
             if value is not None:
-                self.rate('ecs.fargate.cpu.user', value, tags)
+                self.rate('ecs.fargate.cpu.user', value_total, tags)
+            
+            cpu_percent = 0.0
+
+            cpu_delta = float(value_total) - float(container_stats['precpu_stats']['cpu_usage']['total_usage'])
```

Comment:
These direct dict accesses scare me.. can you please use `.get` and wrap in conditional?